raven/src/main/java/org/opencadc/raven/ServiceAvailability.java

```diff
@@ -125,22 +127,27 @@ public AvailabilityStatus getStatus() {
         String note = "service is accepting requests";
         
         try {
-            // check other services we depend on
-            RegistryClient reg = new RegistryClient();
-            String url;
-            LocalAuthority localAuthority = new LocalAuthority();
-
             // check for a certficate needed to perform network ops
             CheckCertificate checkCert = new CheckCertificate(SERVOPS_PEM_FILE);
             checkCert.check();
             log.info("cert check ok");
 
-            CheckResource checkResource;
-            URI credURI = localAuthority.getServiceURI(Standards.CRED_PROXY_10.toString());
-            url = reg.getServiceURL(credURI, Standards.VOSI_AVAILABILITY, AuthMethod.ANON).toExternalForm();
-            checkResource = new CheckWebService(url);
+            String url = "https://localhost.cadc.dao.nrc.ca/baldur/availability";
```

Comment:
What is this? It can' stay in here like this... there are potentially multiple grant providers to call and (for reasons) I would not make this check depend on any of them.

revert changes to this file

Reply:
This is garbage from early debugging of my local system. I was pretty sure I've reverted, not sure how it got in. Fixed now.